probe_scraper/scrapers/moz_central_scraper.py

```diff
@@ -157,6 +159,11 @@ def add_result(ptype, disk_path):
         if uri in error_cache:
             continue
 
+        # This file exists in a bunch of versions, but we only care for it
+        # since firefox 71 (bug 1578661).
+        if rel_path == 'devtools/shared/css/generated/properties-db.js' and int(version) < 71:
```

Comment:
Can we move this to some sort of constant at the top? e.g. `MIN_VERSIONS`

Reply:
I added a function that is called from the tests too but if you prefer a global map that's fine as well.